### PR TITLE
Find-RoleCapability - fixed empty parameter placeholders, copyedits

### PR DESCRIPTION
**Pull request by davidsmatlak**
Version(s) of document impacted
------------------------------
- [ ] Impacts 7 document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Find-RoleCapability Acrolinx scores:
Original version: 93
Updated version: 99

- Replaced empty parameter placeholders with content
- Copyedits and style

### CODE REVIEW

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[c65d86b](https://github.com/davidsmatlak/PowerShell-Docs/pull/4403/commits/c65d86b7a3d00e26bb992d0f14601b3a8231b7bb)_: 

:clock10: **Full build**: the average full build time is _12 min(s) 8 sec(s)_, this is based on the last _97_ build(s) for this repository.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[c65d86b](https://github.com/davidsmatlak/PowerShell-Docs/pull/4403/commits/c65d86b7a3d00e26bb992d0f14601b3a8231b7bb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.1/PowershellGet/Find-RoleCapability.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-3740findrole/reference/5.1/PowershellGet/Find-RoleCapability.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/find-rolecapability?view=powershell-5.1&branch=pr-en-us-4403) (powershell-5.1) |
[reference/6/PowerShellGet/Find-RoleCapability.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-3740findrole/reference/6/PowerShellGet/Find-RoleCapability.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/find-rolecapability?view=powershell-6&branch=pr-en-us-4403) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C5%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201906051447594504-4403%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ZKtyCqyEwwyecA3HVzIFyOeWHKIFHiMb1l9NqBt%2BQxc%3D&st=2019-06-05T14%3A54%3A43Z&se=2019-07-06T14%3A59%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[db3568d](https://github.com/davidsmatlak/PowerShell-Docs/pull/4403/commits/db3568d0152cda14f14da9944204913b141fec6d)_: 

:clock10: **Full build**: the average full build time is _12 min(s)_, this is based on the last _98_ build(s) for this repository.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[db3568d](https://github.com/davidsmatlak/PowerShell-Docs/pull/4403/commits/db3568d0152cda14f14da9944204913b141fec6d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.1/PowershellGet/Find-RoleCapability.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-3740findrole/reference/5.1/PowershellGet/Find-RoleCapability.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/find-rolecapability?view=powershell-5.1&branch=pr-en-us-4403) (powershell-5.1) |
[reference/6/PowerShellGet/Find-RoleCapability.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-3740findrole/reference/6/PowerShellGet/Find-RoleCapability.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/find-rolecapability?view=powershell-6&branch=pr-en-us-4403) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C5%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201906051517068984-4403%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=gjtf5Nl9%2B9VJIL5CzV07AoG2tFW9SyKNEd4QoHCjj44%3D&st=2019-06-05T15%3A24%3A04Z&se=2019-07-06T15%3A29%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/PowerShellGet/Find-RoleCapability.md [hunk 1]
```diff
@@ -304,12 +334,26 @@ Accept wildcard characters: False
...
 This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
 -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
--WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](../Microsoft.PowerShell.Core/About/about_CommonParameters.md).
+-WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
 
 ## INPUTS
 
 ## OUTPUTS
 
+### **PSGetRoleCapabilityInfo**
```
Comment (sdwheeler) [inline reference/6/PowerShellGet/Find-RoleCapability.md]: Can't use bold (or other markup) in a heading.

File: reference/5.1/PowershellGet/Find-RoleCapability.md [hunk 2]
```diff
@@ -273,14 +331,29 @@ Accept wildcard characters: False
...
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
+-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
+-WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
 
 ## INPUTS
 
 ## OUTPUTS
 
+### **PSGetRoleCapabilityInfo**
```
Comment (sdwheeler) [inline reference/5.1/PowershellGet/Find-RoleCapability.md]: Can't use bold (or other markup) in a heading.

File: reference/6/PowerShellGet/Find-RoleCapability.md (hunk 1 above)
Comment (davidsmatlak) [reply reference/6/PowerShellGet/Find-RoleCapability.md]: Thanks, resolved.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[f79a5e5](https://github.com/davidsmatlak/PowerShell-Docs/pull/4403/commits/f79a5e5c146fb787ec09930728f05289ffca30cf)_: 

:clock10: **Full build**: the average full build time is _11 min(s) 47 sec(s)_, this is based on the last _100_ build(s) for this repository.

File: reference/5.1/PowershellGet/Find-RoleCapability.md (hunk 2 above)
Comment (davidsmatlak) [reply reference/5.1/PowershellGet/Find-RoleCapability.md]: Thanks, resolved.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[f79a5e5](https://github.com/davidsmatlak/PowerShell-Docs/pull/4403/commits/f79a5e5c146fb787ec09930728f05289ffca30cf)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.1/PowershellGet/Find-RoleCapability.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-3740findrole/reference/5.1/PowershellGet/Find-RoleCapability.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/find-rolecapability?view=powershell-5.1&branch=pr-en-us-4403) (powershell-5.1) |
[reference/6/PowerShellGet/Find-RoleCapability.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-3740findrole/reference/6/PowerShellGet/Find-RoleCapability.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/powershellget/find-rolecapability?view=powershell-6&branch=pr-en-us-4403) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C5%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201906051607033651-4403%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=IsdLqbDxSSXN%2FxqQuhVTgy4CrPXVZlqkDCA3%2B72Icac%3D&st=2019-06-05T16%3A13%3A35Z&se=2019-07-06T16%3A18%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**